docs/explainer.md

```diff
@@ -1,137 +1,933 @@
-# MiniApp UI Components Explainer 
+# MiniApp Components Explainer 
+
 ## Authors
-Zitao Wang (Huawei)
+
+- Zitao Wang (Huawei)
+- Martin Alvarez (Huawei)
+
 ## 1. Introduction
 
 ### What is this?
-MiniApps are made of high-level building blocks called UI Components, which allow developers to quickly construct the UI for a MiniApp.
+
+MiniApp Components are the building blocks to creating MiniApp pages. Each component encapsulates functionality, data, and styles, enabling developers to create custom and reusable items that can be combined to develop MiniApps. MiniApps include essential components like page containers, textual and multimedia content, and other interactive elements such as forms.
+
+This document aims to specify a set of common practices to define MiniApps Components that allow developers to build similar user interfaces across MiniApp platforms efficiently and provide a consistent user experience.
+
+
 ### Why should we care?
-For MiniApp, there are some pre-standard implementations and wildly used. But these implementations may follow different vendor-specific development guidelines for developing MiniApp UI components. And some of them are defining their own markup languages (WXML, SWAN, AXML), specific event handling, data-binding and rending methods. For example:
 
-- Wechat WXML:
+Although MiniApps Components share the same concept as Web Components, their capabilities and development features differ. The main differences are motivated by the architecture of the existing implementations along the MiniApp ecosystem following the MVVM pattern, based on components developed through domain-specific markup languages and without access to the standard DOM. 
+
+Even though the MiniApp Packaging specification recommends MiniApp agents to support standard Web Components, MiniApp vendors might opt to implement the internal components using the traditional MVVM architecture and, subsequently, manage them through non-standard mechanisms like virtual DOM (VDOM) or content interpolation.
```

Comment:
```suggestion
Even though the MiniApp Packaging specification recommends that MiniApp agents support standard Web Components, MiniApp vendors might opt to implement the internal components using the traditional MVVM architecture and, subsequently, manage them through non-standard mechanisms like virtual DOM (VDOM) or content interpolation.
```

Grammar fix.

Ref: https://english.stackexchange.com/questions/35953/recommend-you-to-vs-recommend-that-you